tool-openssl/x509_test.cc

```diff
@@ -13,6 +13,9 @@ X509* CreateAndSignX509Certificate() {
   bssl::UniquePtr<X509> x509(X509_new());
   if (!x509) return nullptr;
 
+  // Set version to X509v3
+  X509_set_version(x509.get(), 2);
```

Comment:
Why is this change needed? Also please use the constant X509_VERSION_3

Reply:
Noted, changed to using the constant. X509_VERSION_3 adds support for extensions. We use these extensions to configure a self signed certificate.